packages/framework/src/Component/UploadedFile/UploadedFileFactory.php

```diff
@@ -17,4 +33,24 @@ public function create(
     ): UploadedFile {
         return new UploadedFile($entityName, $entityId, $temporaryFilename);
     }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\UploadedFile\Config\UploadedFileEntityConfig $uploadedFileEntityConfig
+     * @param int $entityId
+     * @param string[] $temporaryFilenames
+     * @return \Shopsys\FrameworkBundle\Component\UploadedFile\UploadedFile
+     */
+    public function createUploadedFile(
```

Comment:
is it necessary to add the new method? why do not you just move the required logic into `create` method?

Reply:
oh, they require different arguments... :disappointed: the interface is weird now though

Reply:
...it seems like the `create` method is never called...could it be protected maybe?